lib/commands/new.js

```diff
@@ -64,6 +72,14 @@ module.exports = Command.extend({
       return Promise.reject(new SilentError(message));
     }
 
+    // [WIP] ember new foo --lang
+    let langDef = this.availableOptions.find((opt) => opt.name === 'lang').default;
+    let langInfo = getLangArgInfo(commandOptions.lang);
+    if (langInfo.langArgValue !== langDef && !langInfo.result) {
+      return Promise.reject(new SilentError(langInfo.message));
+    }
+    commandOptions.lang = langInfo.langArgValue;
```

Comment:
Same comment as above, though I don't understand why we need the logic in _both_ `commands/init.js` and `commands/new.js` (since `commands/new.js` is directly invoking `commands/init.js` for the initialization functionality).

Reply:
Removed when consolidating lang-related logic into init in f95d3a9 👍